.github/workflows/build.yml

```diff
@@ -170,9 +170,7 @@ jobs:
   mingw:
     strategy:
       fail-fast: false
-      matrix:
-        arch: ['32', '64']
```

Comment:
Now the matrix has been removed, `fail-fast: false` has no effect and can be removed too. Same with the MSVC job below.

Reply:
a bit late, but here it is.